common/src/test/org/apache/hadoop/hive/common/TestFileUtils.java

```diff
@@ -281,4 +279,32 @@ public void testMakeRelative() {
     relativePath = FileUtils.makeRelative(parentPath, childPath);
     assertEquals(childPath.toString(), relativePath.toString());
   }
+
+  @Test
+  public void testListStatusIterator() throws Exception {
+
+    MockFileSystem fs = new MockFileSystem(new HiveConf(),
+            new MockFileSystem.MockFile("mock:/tmp/.staging", 500, new byte[0]),
+            new MockFileSystem.MockFile("mock:/tmp/_dummy", 500, new byte[0]),
+            new MockFileSystem.MockFile("mock:/tmp/dummy", 500, new byte[0]));
+    Path path = new MockFileSystem.MockPath(fs, "/tmp");
+    RemoteIterator<FileStatus> it = FileUtils.listStatusIterator(fs, path, FileUtils.HIDDEN_FILES_PATH_FILTER);
+    Assert.assertEquals(1, getFileCount(it));
+    RemoteIterator<LocatedFileStatus> itr = FileUtils.listFilesIterator(fs, path, true, FileUtils.HIDDEN_FILES_PATH_FILTER);
+    Assert.assertEquals(1, getFileCount(itr));
```

Comment:
there is already a static import for assertEquals in the class, no need to prefix Assert.